great_expectations/data_context/data_context/serializable_data_context.py

```diff
@@ -413,31 +414,43 @@ def set_ge_config_version(
     @classmethod
     def _find_context_yml_file(
         cls, search_start_dir: Optional[PathStr] = None
-    ) -> Optional[str]:
+    ) -> str | None:
         """Search for the yml file starting here and moving upward."""
-        yml_path = None
         if search_start_dir is None:
-            search_start_dir = os.getcwd()  # noqa: PTH109
+            search_start_dir = pathlib.Path().absolute()
```

Comment:
```suggestion
            search_start_dir = pathlib.Path.cwd().absolute()
```
Do you want the working directory here?